### PR TITLE
Rs 2022.10.02

### PR DESCRIPTION
**Pull request by rschlaikjer**
2022-10-02-124155_1920x1140_scrot

### CODE REVIEW

File: script/Ciel/Day 8/08_01_CIEL08_1.txt [hunk 1]
```diff
@@ -351,7 +375,8 @@ There's probably not a single person in this city that can beat Arcueid.
 [sha:53a7298a0c5e97418080321352ad3b3bf220953e]{
 -- Page 26, Offset 18327.
 -- それではわたしも志貴さま同様、昨夜と同じ業務を行わせていただきます。どうか、お早いお帰りを」
-In that case, I shall conduct my duties as I did last night as well. I pray for your early return."
+In that case, I too shall act as I did last night.
```
Comment (Petrikow) [inline script/Ciel/Day 8/08_01_CIEL08_1.txt:378]: I don't really see the case for the simplification here.

>In that case, I too shall perform my duties as I did last night.

Might be a good middle ground.

File: script/Ciel/Day 8/08_01_CIEL08_1.txt [hunk 2]
```diff
@@ -396,37 +422,39 @@ Now that it's autumn, the tap water's gotten colder.
...
 -- Page 31, Offset 18340.
 -- 「―――誰だ？」
-"―――Who?"
+"―――Who's that?"
 }
 [sha:e2dde82e662f216f1e435bba249859c02687846d]{
 -- Page 31, Offset 18341.
 -- 一瞬、鏡に映った自分に違和感を覚えてしまった。
-For an instant, I feel a sense of discomfort while looking at my reflection.
+I feel a brief sense of alienation when confronted with my reflection in the mirror.
```
Comment (Petrikow) [inline script/Ciel/Day 8/08_01_CIEL08_1.txt:452]: 'alienation' seems like an odd expression to use in a situation like this, and certainly is a very specific translation for 違和感.

> I briefly feel something is off with the reflection that meets me in the mirror.

Is probably more closely what is meant. Any sentence using 'feel something is off' is probably better.

File: script/Ciel/Day 8/08_01_CIEL08_1.txt [hunk 3]
```diff
@@ -41,27 +41,30 @@ My temple is still throbbing with pain, but not to the point that it prevents me
...
-"No, I feel fine. It's more like I was feeling too good yesterday.
+// Reading 良すぎた as just 'better than fine', as in he felt unusually an-anemic (...emic?) yesterday, rather that '_too_ fine'
+"No, this is normal.
+ I was just feeling better than normal yesterday, that's all.
 }
 [sha:42ddd1c2f7dbccdc559c3b0e76c2d1aaa381e37d]{
 -- Page 3, Offset 18267.
 -- 貧血気味なのはいつもの事だから、気にしないでくれ」
-I always feel a bit anemic, so don't worry about it."
+I'm always a little anemic, so don't worry too much about it."
```
Comment (Avuxo) [inline script/Ciel/Day 8/08_01_CIEL08_1.txt:56]: worry about it too much

File: script/Ciel/Day 8/08_01_CIEL08_1.txt [hunk 4]
```diff
@@ -41,27 +41,30 @@ My temple is still throbbing with pain, but not to the point that it prevents me
...
 }
 [sha:c91b7ef2db7485be07a1a6c5c9b377d68d572e5a]{
 -- Page 4, Offset 18268.
 -- 軽く頭を振ってベッドから出る。
-I lightly shake my head and get up from the bed.
+I lightly shake my head and get out of bed.
```
Comment (Avuxo) [inline script/Ciel/Day 8/08_01_CIEL08_1.txt:61]: bars

File: script/Ciel/Day 8/08_01_CIEL08_1.txt [hunk 5]
```diff
@@ -111,7 +120,7 @@ Hmm...?
 [sha:698fbe9568d80febb50de8f0a65bea159a611d5c]{
 -- Page 9, Offset 18279.
 -- 翡翠は意味ありげにこちらを見ている。
-Hisui looks at me with a meaningful look on her face.
+Hisui is staring at me with a meaningful look on her face.
```
[the remark this answers is not in the document]
Comment (Avuxo) [reply script/Ciel/Day 8/08_01_CIEL08_1.txt:123]: meaningful is kinda awkward, sure, but "pregnant" is vetoed

File: script/Ciel/Day 8/08_01_CIEL08_1.txt [hunk 6]
```diff
@@ -161,17 +175,18 @@ Nee-san does it at nine o'clock, then it's my turn at eleven. At one in the morn
 [sha:8cdacdcfb3019238d88fb30bdc75abbb0dea32ea]{
 -- Page 13, Offset 18289.
 -- そこまで説明されて、翡翠が何を言いたがっているのか理解できた。
-Now that she's said this much, I grasp what Hisui's trying to say.
+Now that she's said this much, I grasp what Hisui's driving at.
```
[the remark this answers is not in the document]
Comment (Avuxo) [reply script/Ciel/Day 8/08_01_CIEL08_1.txt:178]: "getting at" is the american version and "insinuating" is like almost formal register

File: script/Ciel/Day 8/08_01_CIEL08_1.txt [hunk 7]
```diff
@@ -216,7 +233,8 @@ It must have been humiliating, saying things that sounded like she was criticizi
 [sha:f3faec426238f6db757af346bcd00197a4c248a3]{
 -- Page 17, Offset 18300.
 -- 翡翠はぜんぜん悪くないよ。もともと夜中に出歩く俺に問題があるんだから。
-You haven't done anything wrong at all, Hisui. I'm the problem here, walking around in the middle of night.
+You haven't done anything wrong at all, Hisui.
```
[the remark this answers is not in the document]
Comment (Avuxo) [reply script/Ciel/Day 8/08_01_CIEL08_1.txt:236]: i like it but keep the name

**Review by Avuxo — APPROVED** [review]
otsukare

File: script/Ciel/Day 8/08_01_CIEL08_1.txt (hunk 5 above)
[the remark this answers is not in the document]
Comment (Petrikow) [reply script/Ciel/Day 8/08_01_CIEL08_1.txt:123]: maybe just 'poignant'?

[the remark this answers is not in the document]
Comment (rschlaikjer) [reply script/Ciel/Day 8/08_01_CIEL08_1.txt:123]: I think that makes it a bit deeper than it is. How about just side translating this as 
> Hisui is starting at me as though there's something {she'd like to say, weighing on her mind}.

?

File: script/Ciel/Day 8/08_01_CIEL08_1.txt (hunk 1 above)
Comment (rschlaikjer) [reply script/Ciel/Day 8/08_01_CIEL08_1.txt:378]: Felt a bit stiff, and I didn't like how it suggested that they were both doing _duties_, though yeah sure I guess you can consider what Shiki's up to a duty. Suggestion works for me.

File: script/Ciel/Day 8/08_01_CIEL08_1.txt (hunk 5 above)
[the remark this answers is not in the document]
Comment (Petrikow) [reply script/Ciel/Day 8/08_01_CIEL08_1.txt]: Yeah, that seems like a good call.

**Pull request merged**